Makefile

```
@@ -1,7 +1,7 @@
 zip: build
-	mv main idp-id-broker-search
+	mv main bootstrap
 	zip idp-id-broker-search.zip idp-id-broker-search
```

Wouldn't this also need to be changed?
```suggestion
	zip idp-id-broker-search.zip bootstrap
```

---

Yes, thank you. Looking at this makes me wonder why this project even has a Makefile.